docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.4'
+
+networks:
+  k6:
+  grafana:
+  timescaledb:
+
+services:
+  timescaledb:
+    image: timescale/timescaledb:latest-pg12
```

Comment:
I needed to change this to `latest-pg12` from `latest` as there is apparently no `latest` anymore